libs/barista-components/form-field/src/form-field.scss

```diff
@@ -13,9 +13,15 @@
 .dt-form-field-body {
   @include dt-form-control-body();
 
-  .dt-focused & {
+  .dt-focused:not(.dt-form-field-non-box) & {
     @include dt-focus-style();
   }
+
+  .dt-form-field-non-box & {
+    border: none;
+    border-radius: 0;
+    min-height: 0;
+  }
```

Comment:
can we move this outside the nesting and write the combined selector here? 
I nearly missed the & and the end - its just hard to read that way

Reply:
👍 changed